model/src/guild/mod.rs

```diff
@@ -25,17 +25,35 @@ mod unavailable_guild;
 mod vanity_url;
 mod verification_level;
 mod widget;
```

Comment:
let's make this module public and move it to the top
```suggestion
pub mod widget;
```